convert2rhel/grub.py

```diff
@@ -324,7 +324,7 @@ def _copy_grub_files(required, optional):
     Return False when any required file has not been copied or is missing.
     """
     if systeminfo.system_info.id != "centos":
-        logger.debug("Skipping copying grub files - only related to CentOS Linux.")
+        logger.debug("Skipping copying GRUB files - only related to CentOS Linux.")
```

Comment:
@mportman12, just a quick question... In the commend above this, we have the word `grub` lowercased (on line 314, more specifically) this shouldn't be uppercase like in this line?

Reply:
Good catch, fixed.